Traceback (most recent call last):
  File "./peepdf.py", line 727, in <module>
    console.cmdloop()
  File "/usr/lib/python2.7/cmd.py", line 142, in cmdloop
    stop = self.onecmd(line)
  File "/usr/lib/python2.7/cmd.py", line 221, in onecmd
    return func(arg)
  File "/home/kali/Downloads/peepdf/PDFConsole.py", line 2858, in do_open
    ret = pdfParser.parse(fileName, forceMode, looseMode)
  File "/home/kali/Downloads/peepdf/PDFCore.py", line 6897, in parse
    sys.exit('Error: Bad PDF header!! (' + versionLine + ')')
SystemExit: Error: Bad PDF header!! ()
Traceback (most recent call last):
  File "./peepdf.py", line 727, in <module>
    console.cmdloop()
  File "/usr/lib/python2.7/cmd.py", line 142, in cmdloop
    stop = self.onecmd(line)
  File "/usr/lib/python2.7/cmd.py", line 221, in onecmd
    return func(arg)
  File "/home/kali/Downloads/peepdf/PDFConsole.py", line 3506, in do_stream
    if self.pdfFile is None:
AttributeError: PDFConsole instance has no attribute 'pdfFile'
Traceback (most recent call last):
  File "./peepdf.py", line 727, in <module>
    console.cmdloop()
  File "/usr/lib/python2.7/cmd.py", line 142, in cmdloop
    stop = self.onecmd(line)
  File "/usr/lib/python2.7/cmd.py", line 221, in onecmd
    return func(arg)
  File "/home/kali/Downloads/peepdf/PDFConsole.py", line 1407, in do_info
    if self.pdfFile is None:
AttributeError: PDFConsole instance has no attribute 'pdfFile'
